File: src/main/java/io/realworld/api/TagsApiImpl.java
package io.realworld.api;

import io.realworld.model.GetTags200Response;

import io.vertx.core.Future;
import io.vertx.ext.web.handler.HttpException;

// Implement this class

public class TagsApiImpl implements TagsApi {
    public Future<ApiResponse<GetTags200Response>> getTags() {
        return Future.failedFuture(new HttpException(501));
    }

}

File: src/main/kotlin/io/realworld/api/TagsApiImpl.kt
package io.realworld.api

import io.realworld.model.response.ApiResponse
import io.realworld.model.response.GetTags200Response
import io.vertx.core.Future
import io.vertx.ext.web.handler.HttpException

class TagsApiImpl : TagsApi {
  override val tags: Future<ApiResponse<GetTags200Response>>
    get() = Future.failedFuture(HttpException(501))
}
